src/tokenizer.rs

```diff
@@ -9,6 +9,7 @@ use crate::cow_rc_str::CowRcStr;
 use crate::parser::ParserState;
 use std::char;
 use std::ops::Range;
+use cssparser_macros::match_byte;
```

Comment:
Instead of adding support for `dummy_match_byte` in the proc macro I think it’d be nicer to keep the cfg-conditional macro_rule and make this import conditional with the opposite cfg

Reply:
Sure, sounds good, done.